code/API_definitions/device-status.yaml

```diff
@@ -32,77 +98,33 @@ servers:
-  - name: Notification Callback
-    description: |
-      Notification received on subscription listener side. 
-      
-      **WARNING**: This endpoint must be exposed on the listener side as POST /notifications 
-      will be posted by the device status resource server to the url provided by notification 
-      listener.
+
 paths:
-  /notifications:
-    post:
-      tags:
-        - Notification Callback
-      summary: "Notification endpoint to notify listener that a subscribed event occurred"
-      description: Notification endpoint to notify listener that a subscribed event occurred. This endpoint must be exposed on the listener side.
-      operationId: postNotification
-      requestBody:
-        required: true
-        content:
-          application/json:
-            schema:
-              $ref: "#/components/schemas/Notification"
-            examples:
-              ROAMING_STATUS:
-                $ref: '#/components/examples/ROAMING_STATUS'
-              ROAMING_CHANGE_COUNTRY:
-                $ref: '#/components/examples/ROAMING_CHANGE_COUNTRY'
-              ROAMING_ON:
-                $ref: '#/components/examples/ROAMING_ON'
-              ROAMING_OFF:
-                $ref: '#/components/examples/ROAMING_OFF'
-              SUBSCRIPTION_ENDS:
-                $ref: '#/components/examples/SUBSCRIPTION_ENDS'
-      responses:
-        204:
-          description: No Content
-        400:
-          $ref: "#/components/responses/Generic400"
-        401:
-          $ref: "#/components/responses/Generic401"
-        403:
-          $ref: "#/components/responses/Generic403"
-        500:
-          description: Server error
-          content:
-            application/json:
-              schema:
-                $ref: "#/components/schemas/ErrorInfo"
-        503:
-          $ref: "#/components/responses/Generic503"
   /roaming:
     post:
       tags:
         - Device roaming status
-      summary: 'Get the current roaming status and the country information'
+      summary: "Get the current roaming status and the country information"
       description: Get the current roaming status and the country information
       operationId: getRoamingStatus
+      security:
+        - threeLegged:
+            - device-status:roaming:read
       requestBody:
         content:
           application/json:
             schema:
-              $ref: '#/components/schemas/RequestRoamingStatus'
+              $ref: "#/components/schemas/RequestRoamingStatus"
         required: true
       responses:
         200:
           description: Contains information about current roaming status
           content:
             application/json:
               schema:
-                $ref: '#/components/schemas/RoamingStatusResponse'
+                $ref: "#/components/schemas/RoamingStatusResponse"
               examples:
                 No-Country-Code:
```

Comment:
Is it example of "No-Country-Code" or "No-Country-Name"
<img width="533" alt="Screenshot 2023-10-25 at 12 12 06" src="https://github.com/camaraproject/DeviceStatus/assets/84573274/13e2b907-581b-47b9-96ea-ccf6f7e9547a">